recipes/sass/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "sass" %}
+{% set version = "3.4.21" %}
+{% set sha256 = "7eeb812f15f67426931d18ed2cda49d6ce73f37d5763720de3519b7f884b7fed" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sass/{{ name }}/archive/{{ version }}.tar.gz
```

Comment:
Proper repo is `https://github.com/sass/node-sass/blob/master/package.json`.

Should maybe rename this package to `node-sass` as well, since there are a bunch of different versions of sass out there.